packages/table/preview/mutableTable.tsx

```diff
@@ -111,11 +124,45 @@ function handleNumberChange(handler: (value: number) => void) {
     return handleStringChange(value => handler(+value));
 }
 
-function getRandomInteger(min: number, max: number) {
+function getRandomObject(propCount: number, depth = 0): object {
+    const childPropCount = propCount;
+    const obj: any = {};
+    while (propCount-- >= 0) {
```

Comment:
nit: I prefer to avoid the `--` and `++` operators, because they're so sneaky and tend to make lines denser.